doc/ssl/SSL_get_version.pod

```diff
@@ -15,6 +15,10 @@ SSL_get_version - get the protocol version of a connection.
 SSL_get_version() returns the name of the protocol used for the
 connection B<ssl>.
 
+The above function should only be called after the intial handshake has been
+completed. Prior to that the results returned from this function may be
+unreliable.
```

Comment:
Why a separate paragraph?  Just put it together into one and refer to `SSL_get_version` as `it` or something similarly appropriate.

Reply:
Fixed.